apps/src/templates/teacherHomepage/SectionsTable.jsx

```diff
@@ -0,0 +1,151 @@
+import React, { PropTypes } from 'react';
+import color from "@cdo/apps/util/color";
+import i18n from '@cdo/locale';
+
+const styles = {
+  table: {
```

Comment:
I drew inspiration from Tanya's spec and a table @Bjvanminnen previously made for progress redesign. There are some things, such as  color.table_header, that already exist and we could probably create a common table style sheet somewhere. I'll leave that until after June 1 if it is a road we want to go down😀